protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -534,6 +534,7 @@ message Tag {
   string description = 2;
   // Additional external documentation for this tag.
   ExternalDocumentation external_docs = 3;
+  map<string, google.protobuf.Value> extensions = 4;
```

Comment:
Please add a comment to explain how to use this (e.g., the convention of using a `x-` prefix for the keys). A link to the extensions page of the openapiv2 spec would be nice.

Reply:
@johanbrandhorst Done!